core/core/src/ansible/roles/deployments/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Include deployments
+  include_tasks: "applications/{{ item.name }}/main.yml"
+  vars:
+    data: "{{ item }}"
+  loop: "{{ kubernetes.deployments }}"
```

Comment:
It seems there is a bug or strange behavior of Ansible: loop is parsed even when condition is not met.
Workaround to consider is:
```
loop: "{{ kubernetes.deployments if kubernetes.deployments else [] }}"
when:
  - kubernetes.deployments is defined
  - kubernetes.deployments
```

Reply:
Will not be changed.